src/creds/acl_add_user.rs

```diff
@@ -0,0 +1,49 @@
+use crate::{crypto::crypto_service::Encryptor, logger::logger_manager::Logger};
+
+use super::cred_manager::{ACLResult, CredsManager, RoleManagement, User, ENCRYPT_KEY, IV_PATTERN};
+
+pub trait IAddUser {
```

Comment:
Do we need `I` prefix for our traits?

Reply:
its global contracts with other programmers like interface in other language it cause more readable for other developers.